client/src/components/payment/payment-complete-modal.vue

```diff
@@ -140,17 +140,19 @@ export default class PaymentCompleteModal extends Mixins(NameRequestMixin, Payme
 
   async completePayment (nrId: number, paymentId: number, action: string) {
     const result: NameRequestPayment = await newRequestModule.completePayment({ nrId, paymentId, action })
-    const paymentSuccess = result.paymentSuccess
+    const paymentSuccess = result?.paymentSuccess
 
     if (paymentSuccess) {
       paymentModule.toggleReceiptModal(true)
-    } else if (!paymentSuccess && result.paymentErrors) {
```

Comment:
This line, and line 143 above, were causing "TypeError: Cannot read property 'paymentSuccess' of undefined."